models/RwdImage.php

```diff
@@ -78,6 +78,32 @@ public function __construct( $attachment ) {
 	}
 
 	/**
+	 * Check if mime type SVG
+	 *
+	 * @return boolean
+	 */
+	public function is_svg_image() {
+		if ( preg_match( '/.svg/',  get_post_mime_type( $this->attachment->ID ) ) ) {
```

Comment:
regular expression is wrong. usual "." symbol means "any character".

we have the whole attachment object actually, so using special func to get mime type is duplicating of queries.

also we have feature where we can overwrite some size with another attachment - in this case this condition is wrong